source/stylesheets/base/_feature_sections.scss

```diff
@@ -21,32 +21,44 @@
 
     .illustration {
       align-items: flex-end;
-      justify-content: center;
+      justify-content: flex-end;
     }
 
     .text-and-description {
       @include grid-column(6 of 12);
       align-items: center;
       display: flex;
     }
+
+    h1 {
+      font-size: 3.1em;
+    }
+
+    time {
+      margin-bottom: 10px;
+    }
   }
 }
 
 .featured-flavors {
-  background: url("../../images/linedpaper.png");
+  @include clearfix;
+  background-color: $white;
 
   .container {
-    height: 500px;
     padding: 50px;
     text-align: center;
 
     .flavors {
       align-items: flex-end;
       display: flex;
+    }
 
-      .flavor {
-        @include grid-column(4 of 12);
-      }
+    .flavor {
+      @include grid-column(4 of 12);
```

Comment:
So this is an interesting case. You're basically using both flexbox and floats inside your `.flavors` component. I think you can do away with the float stuff and just rely on flexbox. That would mean deleting line 57, adding in a justify content on the parent, and setting a flex-basis on each child. We can pair on this if you want.

Reply:
awesome! that's a lot better, thanks.